qcodes/instrument_drivers/rigol/DG1062.py

```diff
@@ -182,8 +187,15 @@ def __init__(self, parent: 'DG1062', name: str, channel: int) ->None:
 
         self.add_parameter(
             "state",
-            set_cmd=f"OUTPUT{channel}:STATE {{}}",
             get_cmd=f"OUTPUT{channel}:STATE?",
+            set_cmd=f"OUTPUT{channel}:STATE {{}}",
+        )
+        
+        self.add_parameter(
```

Comment:
Would you mind adding a `label`, `unit`, and `docstring` to this parameter? (this is done by adding new arguments with these names to the `add_parameter` call, see how it's done in other drivers, even for some parameters in this driver)

Reply:
duty cycle does not have a unit, writing up doc_string: done